### PR TITLE
Launch bad timeout

### PR DESCRIPTION
**Pull request by jimenez**
Adding suicide logic for tasks so as to prevent false timeout for tasks having a long image pull
depends on #1635

### CODE REVIEW

File: test/integration/mesos/api/run.bats [hunk 1]
```diff
@@ -64,3 +64,19 @@ function teardown() {
...
+	# make sure no container exist
+	run docker_swarm ps -qa
+	[ "${#lines[@]}" -eq 0 ]
+
+	# run
+	docker_swarm run -m 20m -d --name test_container busybox sleep 100
+
+	# verify, container is running
+	[ -n $(docker_swarm ps -q --filter=name=test_container --filter=status=running) ]
+}
```
Comment (abronan) [inline test/integration/mesos/api/run.bats]: no newline :ghost:

Comment (jimenez) [inline test/integration/mesos/api/run.bats]: fixed :)

**Comment by jimenez** [comment]
ping @abronan @vieux  Thanks a lot for reviewing this, last PR I'd like to get in before the code freeze.

**Comment by abronan** [comment]
LGTM

thanks @jimenez ;)

**Comment by vieux** [comment]
LGTM, waiting for tests to be green on stable

**Pull request merged**